src/app/components/common/node-editor/filter.ts

```diff
@@ -2,24 +2,35 @@ import { ClassicPreset } from "rete";
 import { EnabledControl, NameControl, PluginControl } from "./controls/common-custom-control";
 import { FilterColorControl } from "./controls/filter-custom-control";
 
+export class PseudoNodeControl extends ClassicPreset.Control {
+  constructor(public pseudoConnection: boolean) {
+    super();
+  }
+}
 
 export class Filter extends ClassicPreset.Node {
   height = 90;
   width = 198;
   parent?: string;
+  pseudoNode?: boolean;
```

Comment:
Is `pseudoNode` that filter node which is withOUT connection on the canvas?

Reply:
Yes